ts/features/walletV3/payment/screens/WalletPaymentPickMethodScreen.tsx

```diff
@@ -260,7 +243,7 @@ const WalletPaymentPickMethodScreen = () => {
     }
   }, [isLoading, genericMethodsListItems, savedMethodsListItems]);
 
-  if (!isLoading && savedMethodsListItems.length === 0) {
+  if (pot.isSome(userWalletsPots) && userWalletsPots.value.length === 0) {
```

Comment:
why this change? since `savedMethodListItems` already maps all non-some cases to `[]`, the old solution on paper seems better to me.

Reply:
`isLoading` does not change at the same time this `pot.isLoading` changes. There is a couple of frames where the component is rendered when it should not, causing a bug.